setup.py

```diff
@@ -29,13 +28,11 @@
 with open("README.md", "r", encoding="utf-8") as fh:
     long_description = fh.read()
 
-NoneType = type(None)
 
-
-def extractall(zip: typing.Any, path: str) -> NoneType:
+def extractall(zip: zipfile.ZipFile, path: str) -> None:
     for name in zip.namelist():
         member = zip.getinfo(name)
-        extracted_path = zip._extract_member(member, path, None)
+        extracted_path = zip.extract(member, path, None)
```

Comment:
```suggestion
        extracted_path = zip.extract(member, path)
```
nit: None is not needed there.